### ISSUE TITLE
Suggestion: Implement a way to read all warnings and errors and make it easier to solve them

### ISSUE THREAD

**Issue by Chunnyluny** [RimSort/RimSort#439]
**Is your feature request related to a problem? Please describe.**

- If you have a lot of warnings or errors, it is hard to see in the tooltip at the bottom.

**Describe the solution you'd like**

- It would be good if you can click on the warnings/errors, and it will pop up a window, which you can minimalize, if you are working on fixing the errors/replacing outdated mods it would be nice that they can be ticked off, after refreshing. I think it makes sense to have the window minimalized so you can work on the errors, because you can only see the warnings by mouseover hovering.
- If you have a lot of warnings, it could also be possible to make them scrollable. 
- Another possibility could be if a mod misses another mod. It summarizes it like, for example, some mods that have warnings/errors and have all in common that they miss harmony.pardeike that it would be better to list it as missing mod and not list it as missing mod for all those separate mods, which would clutter the list. 
- Another suggestion that I have is, when you click on the warning/errors at the bottom, it would be useful to pop up a window that will prompt you to download the missing mods, and you only need to confirm if you want to download it with steam or steam CMD. It would also be useful if you could see the preview of the mods that you are going to download as thumbnails, otherwise it could be possible that you download the wrong version. You can download by name, but you can recognise a preview from the steam page to see if it is the correct version of the mod.  

**Describe alternatives you've considered**

- These suggestions will improve the warnings/errors toggles, and I have checked if there is any duplicate feature request and I couldn't find one that is detailed as mine.

**Additional context**
I will leave it here and come up with a QT preview screenshot/video if I'm able to express my ideas in the program. If you understand what I have in mind, you can make something which I can give my feedback to, to confirm if it is similar to my ideas.

**Comment by Chunnyluny** [comment]
@hk21702 @LionelColaso @twstagg

**Comment by Raizanad** [comment]
As I'm working on making my list at #406 into separate issues, I want to add to this discussion as a suggestion the feature request I was going to make as a standalone issue:

>  Feature: Sort to the top mods that have errors and warnings.

**Comment by LionelColaso** [comment]
Fixed in #732
